### PR TITLE
Adds favicons for the website

### PR DESCRIPTION
**Pull request by codefromthecrypt**
This adds favicons and also Web Manifest configuration for
https://func-e.io. This comes with great thanks to @peterj,
who contributed the png and ico files.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -37,7 +37,7 @@ bin $(BIN): $(GORELEASER)
 site: $(HUGO)
 	@echo "--- site ---"
 	@git submodule update
-	@cd site && $(HUGO) server -D
+	@cd site && $(HUGO) server --disableFastRender -D
```
Comment (codefromthecrypt) [inline Makefile:40]: I added this to prevent weird glitches when you update hugo between commits

**Comment by codefromthecrypt** [comment]
netlify preview URL is https://deploy-preview-323--func-e.netlify.app/

**Comment by mathetake** [comment]
Oh netlify bots stopped putting preview links completely? 😄

**Review by mathetake — APPROVED** [review]

**Comment by codefromthecrypt** [comment]
> Oh netlify bots stopped putting preview links completely? 😄
yes I think we offended them :D

**Comment by mathetake** [comment]
btw I don't think we need to run unittests on changes under site/ 😄

**Comment by codefromthecrypt** [comment]
> btw I don't think we need to run unittests on changes under site/ 😄

I think usually we wouldn't but I couldn't resist upgrading hugo, which tripped the root path

**Comment by mathetake** [comment]
ah - ic 😄

**Pull request merged**